### ISSUE TITLE
Feature request: format `_text_` as italics

### ISSUE THREAD

**Issue by waldekmastykarz** [StfBauer/markshell#17]
Would it be possible to format `_text_` as italics or highlight it in a different way rather than outputting raw markdown?

<img width="412" alt="image" src="https://user-images.githubusercontent.com/11164679/91637205-21be1f00-ea07-11ea-9828-ccf710e203a5.png">

### Repro

Markshell https://github.com/waldekmastykarz/cli-microsoft365/blob/552d9fd2b83f8c3487e1d8b274b4965b6275ec78/docs/manual/docs/cmd/spo/site/site-add.md

**Comment by StfBauer** [comment]
Well actually it was there but for some reason was removed during "optimisation". Now I potentially found a better way to regex replace all styling elements.

<img width="209" alt="image" src="https://user-images.githubusercontent.com/5503835/91645968-b007c480-ea4a-11ea-85d1-689f019e5473.png">

Seems like by requesting the found groups from the regex a lot of problems are solved.

```javascript
const _highlightText = (content, regexMatch, colorFunction) => {

    let match;

    while ((match = regexMatch.exec(content)) !== null) {
        
        // This is necessary to avoid infinite loops with zero-width matches
        if (match.index === regexMatch.lastIndex) {
            regexMatch.lastIndex++;
        }

        content = content.replace(match[0], colorFunction(match[1]));

    }

    return content;

}
```

**Comment by StfBauer** [comment]
Fixed in 📦  markshell@0.0.6-beta3

**Comment by waldekmastykarz** [comment]
Confirmed fixed
